From 6c93035fad4e14c9973362affb64914bb93a95ae Mon Sep 17 00:00:00 2001
From: Frederic Danis <frederic.danis@linux.intel.com>
Date: Wed, 23 Sep 2015 18:18:10 +0000
Subject: [PATCH] Bluetooth: Remove useless rx_lock spinlock

---
 drivers/bluetooth/hci_ldisc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/bluetooth/hci_ldisc.c b/drivers/bluetooth/hci_ldisc.c
index 116255ad5661..cd0297b6e245 100644
--- a/drivers/bluetooth/hci_ldisc.c
+++ b/drivers/bluetooth/hci_ldisc.c
@@ -489,7 +489,6 @@ static int hci_uart_tty_open(struct tty_struct *tty)
 	INIT_WORK(&hu->init_ready, hci_uart_init_work);
 	INIT_WORK(&hu->write_work, hci_uart_write_work);
 
-	spin_lock_init(&hu->rx_lock);
 	mutex_init(&hu->proto_lock);
 
 	/* Flush any pending characters in the driver and line discipline. */
